static/js/containers/DashboardPage.js

```diff
@@ -119,14 +111,16 @@ class DashboardPage extends React.Component {
     dispatch:                 Dispatch,
     setCalculatorVisibility:  (b: boolean) => void,
     ui:                       UIState,
-    courseTeamEmail:          EmailState,
+    email:                    AllEmailsState,
     documents:                DocumentsState,
     fetchDashboard:           () => void,
     orderReceipt:             OrderReceiptState,
     courseEnrollments:        CourseEnrollmentsState,
     financialAid:             FinancialAidState,
     location:                 Object,
     pearson:                  PearsonAPIState,
+    emailDialog:              React$Element<*>|null,
+    emailOpenActions:         {[emailType: string]: () => void}
```

Comment:
yeah, this was done on purpose. we already have use cases for multiple email types in the same page. passing them in as an object that keys on the email type was the best way i could think of to do that. the export line will look something like this:

```javascript
export default R.compose(
  connect(mapStateToProps),
  withEmailDialog(COURSE_EMAIL_TYPE),
  withEmailDialog(LEARNER_EMAIL_TYPE)
)(DashboardPage);
```

`emailOpenActions` in that case will look like this:

```javascript
{
  'courseTeamEmail': //course team email open function,
  'learnerEmail': //learner-to-learner email open function,
}
```

The HOC adds to that object with each invocation